sycl/doc/extensions/proposed/sycl_ext_oneapi_graph.asciidoc

```diff
@@ -844,6 +850,34 @@ Exceptions:
 * Throws synchronously with error code `invalid` if the topology of `graph` is
   not the same as the existing graph topology, or if the nodes were not added in
   the same order.
+
+  |
+[source, c++]
+----
+using namespace ext::oneapi::experimental;
+template <typename T>
+void command_graph<graph_state::executable> 
+update_parameter(const T& from, const T& to);
+----
+
+|Updates a parameter across the executable graph. All nodes which
```

Comment:
Would tweak this description slightly, so we say what the entry-point does in a term other than "update":

E.g say "replace" - "Updates a parameter across the executable graph such that all nodes which use `from` as a parameter have the parameter replaced by `to`"